lib/yt/models/claim.rb

```diff
@@ -14,6 +14,12 @@ def initialize(options = {})
         @asset = options[:asset] if options[:asset]
       end
 
+      def update(attributes = {})
```

Comment:
Will you add some method documentation on top for a public method? that would be nice

then I think this part should be included, to avoid surprises
> If you are submitting an update request, and your request does not specify a value for a property that already has a value, the property's existing value will be deleted.

Reply:
updated